webapp/src/main/java/rocks/metaldetector/service/notification/messaging/NotificationScheduler.java

```diff
@@ -0,0 +1,73 @@
+package rocks.metaldetector.service.notification.messaging;
+
+import lombok.AllArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import rocks.metaldetector.butler.facade.dto.ReleaseDto;
+import rocks.metaldetector.persistence.domain.notification.NotificationConfigEntity;
+import rocks.metaldetector.persistence.domain.notification.NotificationConfigRepository;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static java.time.temporal.ChronoUnit.WEEKS;
+
+@Component
+@AllArgsConstructor
+public class NotificationScheduler {
+
+  private final NotificationConfigRepository notificationConfigRepository;
+  private final NotificationReleaseCollector notificationReleaseCollector;
+  private final NotificationSenderSupplier notificationSenderSupplier;
+
+  @Scheduled(cron = "0 0 4 * * SUN")
```

Comment:
should we align this to 7?

```suggestion
  @Scheduled(cron = "0 0 7 * * SUN")
```